client_cli/pytests/README.md

```diff
@@ -1,92 +1,164 @@
+- `pytest.ini` — configuration file for the `pytest` framework.
 
-```
-CLIENT_CLI_DIR=/path/to/iroha_client_cli/with/config.json/dir/
-TORII_API_PORT_MIN=8080
-TORII_API_PORT_MAX=8083
-```
-Replace `/path/to/iroha_client_cli/dir` with the actual paths to the respective files on your system.
+All tests are written with [Allure Report](https://allurereport.org/) in mind, and therefore require certain configuration prior to being executed.\
+For details, see [Running Tests](#running-tests) and [Viewing Test Reports](#viewing-test-reports).
 
-If the `.env` file is not present or these variables are not defined in it
+## Iroha 2 Test Model
 
-## Poetry Configuration
+The Iroha 2 Test Model consists of several test categories that cover different aspects of the Iroha 2 blockchain platform.\
+The test model has the following structure:
+
+- **Configurations**: Test configurations for the Iroha 2 platform.
+
+---
+
+- **Accounts**: Test cases for account-related operations, such as account registration, key management, and metadata manipulation.
+- **Assets**: Test cases for asset-related operations, including asset creation, minting, burning, transferring, and managing asset definitions and metadata.
+- **Atomicity**: Test cases for transaction atomicity, including multiple instructions within a single transaction, paired instructions, and invalid instructions.
+- **Domains**: Test cases for domain-related operations, such as registering and unregistering domains.
+- **Roles**: Test cases for roles management.
+
+---
+ALT:
+- **Accounts**: Test cases for account-related operations:
+	- `test_accounts_query_filters.py` — various accounts-related filter queries.
+	- `test_register_accounts.py` — account registration.
+	- `test_set_key_value_pair.py` — key management.
+- **Assets**: Test cases for asset-related operations:
+	- `test_assets_query_filters.py` — various assets-related filter queries.
+	- `test_burn_assets.py` — burning assets.
+	- `test_mint_assets.py` — minting assets.
+	- `test_register_asset_definitions.py` — registering various types of asset definitions.
+	- `test_transfer_assets.py` — transferring assets.
+- **Atomicity**: Test cases for transaction atomicity:
+	- `test_multiple_instructions_within_transaction.py` — multiple instructions within a single transaction.
+	- `test_pair_instructions_within_transaction.py` — paired instructions within a single transaction.
+	- `test_wrong_instructions_within_transaction.py` — invalid instructions within a single transaction.
+- **Domains**: Test cases for domain-related operations:
+	- `test_domains_query_filters.py` — various domain-related filter queries.
+	- `test_register_domains.py` — registering various types of domains.
+	- `test_transfer_domains.py` — transferring a domain.
+- **Roles** Test cases for roles management:
+	- `test_register_roles.py` — registering a role, attaching permissions to a role, granting a role to an account.
```

Comment:
I think it is too detailed.

Reply:
I still opted to use the full version for the sake of clarity and straightforward visibility.